org.adempiere.base/src/org/compiere/model/SystemProperties.java

```diff
@@ -54,6 +54,7 @@ public class SystemProperties {
 	private static final String PropertyFile = "PropertyFile";
 	private static final String PropertyHomeFile = "PropertyHomeFile";
 	private static final String TestOCI = "TestOCI";
+	private static final String TRACE_NULL_TRX_CONNECTION = "TRACE_NULL_TRX_CONNECTION";
```

Comment:
This will have very significant impact on performance, need to have warning everywhere.

Reply:
I just added a warning in the isTraceNullTrxConnection() method, and plan to do it in the documentation page too.